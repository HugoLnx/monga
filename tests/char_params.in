int xpto(char a, int b, char c) {
	printf("a: %c\n", a);
	printf("b: %d\n", b);
	printf("c: %c\n", c);
}

int main() {
	xpto('A', 99, 'C');
}
