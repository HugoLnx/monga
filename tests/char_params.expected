a: A
b: 99
c: C
